website/src/styles/_installation.scss

```diff
@@ -0,0 +1,80 @@
+.inst-body {
```

Comment:
`inst` -> `installation`

Reply:
Do you mean changing the `inst-*` classes name to `installation-*`?

Reply:
Yes Sorry, that's what I meant